src/processing/mode/android/AndroidBuild.java

```diff
@@ -179,6 +181,8 @@ public File createProject() throws IOException, SketchException {
 //      InputStream input = PApplet.createInput(getCoreZipLocation());
 //      PApplet.saveStream(new File(libsFolder, "processing-core.jar"), input);
       Base.copyFile(coreZipFile, new File(libsFolder, "processing-core.jar"));
+      
+      copySupportV4(libsFolder);
```

Comment:
Alright, let's do it. Let's update the code to use API 15 as our minimum: https://github.com/processing/processing-android/issues/119